schemas/Case.json

```diff
@@ -30,8 +30,8 @@
         "additionalProperies": false
       }
     },
-    "transfers": {
-      "description": "URIs of transfers that are related to this case - in the future this feature should be replaced by a generic crypto-event subscription system",
+    "completion_targets": {
+      "description": "URIs of completion_targets that are invoked on the completion of this case",
```

Comment:
I think it would be good to specify what "invoked" means here: HTTP PUT request containing ...